### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by LeeKamentsky** [pypi/support#1066]
### PyPI Username

`Lee.Kamentsky`

### Reason for request

Lost 2FA authentication app

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by ewdurbin** [comment]
Contacted via primary email address on the account, please read the instructions and reply to proceed with recovery.

**Comment by LeeKamentsky** [comment]
Hi Ed, the directions you posted request that I add a branch to the python-bioformats project:
```In order to verify the request, please push a branch with the name GtD0uZqm2PREA6x3 to the public source code repository for the python-bioformats project and respond to this email.
```
I have left that team and no longer have access to the repo. Would it be possible to do something similar to https://github.com/LeeKamentsky/python-javabridge?

Thanks,
Lee

**Comment by ewdurbin** [comment]
Sorry for delayed response. It was missed because responses are expected to the email sent to the primary email address on the account, not here on the issue.

Please respond to the email and we can discuss other verification methods.

**Comment by LeeKamentsky** [comment]
Thanks for getting back to me, Ed.

I could push to a branch on
https://github.com/LeeKamentsky/python-javabridge if that would help. Or
post a gist or something.

- Lee
(he, him, his)

**Comment by Thespi-Brain** [comment]
Hi @LeeKamentsky,

I am following up regarding your request. Are you still having an issue regarding your recovery codes and logging in? Let us know. I will keep this issue open for another week unless I hear from you earlier. After that, feel free to create a new request if you are still having trouble. Thanks!

**Comment by LeeKamentsky** [comment]
Yes, it's been several years since I've had access. Can I please use https://github.com/LeeKamentsky/python-javabridge to prove my identity?

**Comment by Thespi-Brain** [comment]
Hi @LeeKamentsky,

We can only initiate account verifications for repos that are directly linked to the user's PyPI account. Currently, python-javabridge's repo is not linked. So, I have chosen another project's public repo that is linked to your account. An email is being sent to the primary email address associated with your account. Please follow the instructions in that email and reply to it, and we will go from there. Thanks!

**Comment by LeeKamentsky** [comment]
The branch should be there:
https://github.com/LeeKamentsky/shelllnk/tree/0XOLxfB8I0c4hOhg
- Lee
(he, him, his)

**Comment by Thespi-Brain** [comment]
Hi @LeeKamentsky,

Thanks for completing the verification process. The verification was successful! Your 2FA has been reset and a password reset has been issued. You should be able to regain access to your account after completing the password reset. Let us know if you have any more issues. Thanks.
